doc/github-complete.txt

```diff
@@ -302,6 +302,16 @@ g:github_complete_github_api_token          *g:github_complete_github_api_token*
 
     Note: Please do not expose your token.  It's secret.
 
+g:github_complete_github_api_url              *g:github_complete_github_api_url*
+
+    Set the base api url with this variable.
+    Default value is "https://api.github.com/".
+
+g:github_complete_github_repo                    *g:github_complete_github_repo*
+
+    Set the github host you're working on with this variable.
+    Default value is "github.com".
+
```

Comment:
I guess you're a user of GitHub Enterprise. Can we have one variable `g:github_complete_ghe_host`? If my understanding is correct, `https://api.{host}` is an API endpoint for the `{host}` so we can configure with one variable. What do you think?
Please let me know if there is a case where API endpoint URL and repository host are different.

Reply:
I forced pushed the branch to have clean history.

In the case of ghe, the API endpoint URL and repository host are different: https://developer.github.com/enterprise/2.14/v3/enterprise-admin/#endpoint-urls

This new commit handles ghe with only one variable, for simplicity's sake, I agree with you.

Let me know if you have any feedback.

Regards,